tests/python/multidevice/test_communication.py

```diff
@@ -5,14 +5,10 @@
 import pytest
 import torch
 
-import fixtures
 import nvfuser
 from nvfuser import DataType, FusionDefinition
 
 
-multidevice_test = fixtures.multidevice_test
```

Comment:
another pytest magic on automatic discovery of `conftest.py`.

Thanks for linking the pytest doc in the PR description.

I wonder what you meant by this vvv, is it just an out-dated comment?
> Import fixtures using from ... import ...

Reply:
Yes. In fact, it's not recommended to import a scope=session fixture because these imports may be recognized as different fixtures (cf. https://github.com/pytest-dev/pytest/issues/1878)